--- configure.in.orig	2005-11-25 20:58:27.000000000 +0100
+++ configure.in	2007-05-15 14:55:26.000000000 +0200
@@ -60,7 +60,7 @@
 else
         AC_DEFINE(HAVE_MYSQL)
         MYSQLINCLUDES=`$d/mysql_config --include`
-        MYSQLLIBS=`$d/mysql_config --libs`
+        MYSQLLIBS=`$d/mysql_config --libs | sed 's/-Wl,--as-needed //'`
 
 	DATABASE_DIR="${DATABASE_DIR} mysql"
 
--- configure.in.orig	2007-05-15 15:02:06.000000000 +0200
+++ configure.in	2007-05-15 15:10:36.000000000 +0200
@@ -83,7 +83,7 @@
 
 	AC_MSG_CHECKING(for mysql_real_escape_string support)
 
-	MYSQL_FUNCTION_TEST=`strings ${MYSQLLIBS}/libmysqlclient.so | grep mysql_real_escape_string`
+	MYSQL_FUNCTION_TEST=`strings $(echo ${MYSQLLIBS} | grep -m 1 -o -- '-L/[[^ ]]*/ ' | sed 's/[[-L ]]//g')/libmysqlclient.so | grep mysql_real_escape_string`
 
 	if test "x$MYSQL_FUNCTION_TEST" = x
 	then
