--- acinclude.m4.orig	2009-03-06 18:54:04.000000000 +0100
+++ acinclude.m4	2009-09-18 20:28:22.201107773 +0200
@@ -207,7 +207,7 @@
 
 AC_ARG_WITH(pcap,
 	[  --with-pcap=PREFIX		Prefix of your libpcap installation],
-	[pcap_prefix=$withval], [pcap_prefix=])
+	[pcap=$withval], [pcap_prefix=])
 AC_ARG_WITH(pcap-inc,
 	[  --with-pcap-inc=PATH		Path to the include directory of pcap],
 	[pcap_inc=$withval], [pcap_inc=/usr/include])
